src/guides/v2.4/graphql/queries/store-config.md

```diff
@@ -301,6 +321,16 @@ Attribute |  Data Type | Description | Example
 `root_category_id` | Int | The ID of the root category
 `title_separator` | String | Identifies the character that separates the category name and subcategory in the browser title bar | `-`
 
+### Supported Catalog attributes
+
+Use the `customer` attributes to retrieve information about the store's customer settings. These attributes are defined in the `CustomerGraphQl` module.
+
+Attribute |  Data Type | Description | Configuration path | Example
```

Comment:
Please, rename the "Example" column to "Default Value" or so, to reflect the actual purpose of the column.

Reply:
**done**